Newest TestFile Github Ver/prototype.usrfnc.js

```diff
@@ -1,9 +1,10 @@
 module.exports = function() {
```

Comment:
This file can be cleared yeah?

Reply:
Depends if things are wrapped in between it
if not than we just need the stuff to be exported so the code can be used at end of the doc, or at the start of doc with everything between the brackets